iOS/AppboyReactBridge/AppboyReactBridge/AppboyReactBridge.m

```diff
@@ -341,6 +362,40 @@ - (void)feedUpdated:(NSNotification *)notification {
   [mainViewController presentViewController:feedModal animated:YES completion:nil];
 }
 
+#pragma mark - Content Cards
+
+- (void)handleContentCardsUpdated:(NSNotification *)notification {
+  BOOL updateIsSuccessful = [notification.userInfo[ABKContentCardsProcessedIsSuccessfulKey] boolValue];
+  if (hasListeners && updateIsSuccessful) {
+    RCTLogInfo(@"contentCardsUpdated sent to the bridge");
+    [self sendEventWithName:kContentCardsUpdatedEvent body:nil];
+  }
+}
+
+- (NSArray *)getMappedContentCards {
+  NSArray<ABKContentCard *> *cards = [[Appboy sharedInstance].contentCardsController getContentCards];
+
+  NSMutableArray *mappedCards = [NSMutableArray arrayWithCapacity:[cards count]];
+  [cards enumerateObjectsUsingBlock:^(id card, NSUInteger idx, BOOL *stop) {
+    [mappedCards addObject:RCTFormatContentCard(card)];
+  }];
+
+  return mappedCards;
+}
+
+- (nullable ABKContentCard *)getContentCardById:(NSString *)idString {
+  NSArray<ABKContentCard *> *cards = [[Appboy sharedInstance].contentCardsController getContentCards];
+  NSPredicate *predicate = [NSPredicate predicateWithFormat:@"idString == %@", idString];
+  NSArray *filteredArray = [cards filteredArrayUsingPredicate:predicate];
+
+  if ([filteredArray count]) {
+    // TODO: is it fair check? Braze should not provide cards with duplicated idString
+    return filteredArray[0];
+  }
+
+  return nil;
```

Comment:
Minor: lines 389-396 can be re-written as:
```
return [[cards filteredArrayUsingPredicate:predicate] firstObject];
```

Reply:
updated